default-theme-CF-version.yml

```diff
@@ -25,7 +25,8 @@ base:
   text_align: justify
   font_color: 333333
   font_family: Noto Serif
-  font_size: 10.5
+  #DCH font_size: 10.5
+  font_size: 10
```

Comment:
```suggestion
  font_size: 10.5
```

(Left-over from development, I guess.)

Reply:
Urgh, yes! I might put in a separate PR for 10 (or even 9.5 :))